sp800-63b/references.md

```diff
@@ -63,52 +63,52 @@
 
 <a name="RFC20"></a>[RFC 20] Cerf, V., *ASCII format for network interchange*, STD 80, RFC 20, DOI 10.17487/RFC0020, October 1969, <https://doi.org/10.17487/RFC0020>.
 
-<a name="RFC5246"></a>[RFC 5246] IETF, *The Transport Layer Security (TLS) Protocol Version 1.2*, RFC 5246, DOI 10.17487/RFC5246, August 2008, available at: <https://doi.org/10.17487/RFC5246>.
+<a name="RFC5246"></a>[RFC 5246] IETF, *The Transport Layer Security (TLS) Protocol Version 1.2*, RFC 5246, DOI 10.17487/RFC5246, August 2008, <https://doi.org/10.17487/RFC5246>.
 
-<a name="RFC5280"></a>[RFC 5280] IETF, *Internet X.509 Public Key Infrastructure Certificate and CRL Profile*, RFC 5280, DOI 10.17487/RFC5280, May 2008, available at: <https://doi.org/10.17487/RFC5280>.
+<a name="RFC5280"></a>[RFC 5280] IETF, *Internet X.509 Public Key Infrastructure Certificate and CRL Profile*, RFC 5280, DOI 10.17487/RFC5280, May 2008, <https://doi.org/10.17487/RFC5280>.
 
-<a name="RFC6238"></a>[RFC 6238] IETF, *TOTP: Time-Based One-Time Password Algorithm*,RFC 6238, DOI 10.17487/RFC6238, available at: <https://doi.org/10.17487/RFC6238>.
+<a name="RFC6238"></a>[RFC 6238] IETF, *TOTP: Time-Based One-Time Password Algorithm*,RFC 6238, DOI 10.17487/RFC6238, <https://doi.org/10.17487/RFC6238>.
 
-<a name="RFC6960"></a>[RFC 6960] IETF, *X.509 Internet Public Key Infrastructure Online Certificate Status Protocol - OCSP*, RFC 6960, DOI 10.17487/RFC6960, available at: <https://doi.org/10.17487/RFC6960>.
+<a name="RFC6960"></a>[RFC 6960] IETF, *X.509 Internet Public Key Infrastructure Online Certificate Status Protocol - OCSP*, RFC 6960, DOI 10.17487/RFC6960, <https://doi.org/10.17487/RFC6960>.
 
 <a name="UAX15"></a>[UAX 15] Unicode Consortium, *Unicode Normalization Forms*, Unicode Standard Annex 15, Version 9.0.0, February, 2016, available at: <http://www.unicode.org/reports/tr15/>.
 
 ### 3. NIST Special Publications
 
 NIST 800 Series Special Publications are available at: <http://csrc.nist.gov/publications/nistpubs/index.html>. The following publications may be of particular interest to those implementing systems of applications requiring e-authentication.
 
-<a name="SP800-38B"></a>[SP 800-38B] NIST Special Publication 800-38B, *Recommendation for Block Cipher Modes of Operation: the CMAC Mode for Authentication*, October, 2016, available at: <http://dx.doi.org/10.6028/NIST.SP.800-38B>.
+<a name="SP800-38B"></a>[SP 800-38B] NIST Special Publication 800-38B, *Recommendation for Block Cipher Modes of Operation: the CMAC Mode for Authentication*, October, 2016, <http://dx.doi.org/10.6028/NIST.SP.800-38B>.
```

Comment:
I'm under the impression that all of the dx.doi.org URLs are now doi.org, but check me on that.